code/game/gamemodes/wizard/wizard.dm

```diff
@@ -164,7 +164,7 @@
 
 	// Wizards
 	for(var/datum/mind/wizard in wizards)
-		if(!iscarbon(wizard.current || wizard.current.stat == DEAD) || istype(wizard.current, /obj/item/mmi)) // wizard is in an MMI, don't count them as alive
+		if(!iscarbon(wizard.current) || wizard.current.stat == DEAD || istype(wizard.current, /obj/item/mmi)) // wizard is in an MMI, don't count them as alive
```

Comment:
tbh if you're checking `iscarbon()` here, I don't think the last `istype()` will really matter since it should already have dropped out

Reply:
Yeah makes sense, removing it